File: JSBASIC/161_count_even_arr.js
// 161	161_count_even_arr.js	Count how many even numbers are present in an array.

let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
let count = 0;
for (let i = 0; i < arr.length; i++) {
    if (arr[i] % 2 === 0) count++;
}
console.log("Array:", arr);
console.log("Count of even numbers:", count);

File: JavaBasic/161_count_even_arr.java
// 161	161_count_even_arr.java	Count how many even numbers are present in an array.
public class _161_count_even_arr {
    public static void main(String[] args) {
        int[] arr = {1,2,3,4,5,6,7,8,9,10};
        int count = 0;
        for (int x : arr) if (x%2==0) count++;
        System.out.println("Even count: " + count);
    }
}
